heareval/tasks/config/nsynth_pitch.py

```diff
@@ -0,0 +1,24 @@
+"""
+Configuration for the nsynth pitch detection task
+"""
```

Comment:
Could we maybe standardize these configs as like a JSON file (or Python dict) to make stuff more generic?

Reply:
Sure -- I'm into something like a json file